platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -0,0 +1,822 @@
+# TSS-Ledger-ID
+
+---
+
+## Summary
+
+This proposal is for the integration of a threshold signature scheme into the consensus nodes to allow them to create a
+private/public key pair for the ledger that can be used to sign blocks and construct block proofs. The private key must
+be a secret that no one knows. The public key is known by all and functions as the ledger id. The consensus nodes must
+be able to aggregate their individual signatures on a message into a valid ledger signature for the message that is
+verifiable by the ledger id. Through signing a block with the ledger private key, we can use Merkle Proofs to
+extend the signature to cover the contents of the block and the state changes processed in that round. This forms
+the basis of Block Proofs and provides a way for external parties to verify that certain facts are true on the ledger
+at a particular point in time. Part of our goal is that users could write EVM smart contracts that can verify the
+Block Proofs and make decisions based on their claims.
+
+| Metadata           | Entities                                   | 
+|--------------------|--------------------------------------------|
+| Designers          | Edward, Cody, Austin, Kore, Anthony, Artem |
+| Functional Impacts | Services, DevOps                           |
+| Related Proposals  | TSS-Library, TSS-Roster, TSS-Block-Signing |
+| HIPS               | N/A                                        |
+
+---
+
+## Purpose and Context
+
+Users of a ledger want to have cryptographic proofs that a statement was true on the ledger at a particular point in
+time. Our chosen method for providing these types of proofs is to assign a permanent public/private key pair to the
+ledger for use in signing blocks and constructing block proofs. The ledger private key must be a secret that no one
```

Comment:
I think it makes more sense to describe the purpose of this proposal as signing block hashes to prove the integrity of the blocks in a way that only the ledger id is required in order to verify. The fact that block nodes will use these signatures to produce state proofs is another matter.